src/Effects/Effects.UWP/Effects/Picker/PickerChangeColor.cs

```diff
@@ -1,22 +1,22 @@
 ﻿using Windows.UI.Xaml.Controls;
-using FormsCommunityToolkit.Effects.UWP.Effects;
 using Xamarin.Forms;
 using Xamarin.Forms.Internals;
 using Xamarin.Forms.Platform.UWP;
 using Windows.UI.Xaml.Media;
+using FormsCommunityToolkit.Effects;
+using PickerChangeColor = FormsCommunityToolkit.Effects.UWP.Effects.PickerChangeColor;
```

Comment:
these seem to give issues :)

What about
using RoutingEffects = FormCommunityToolkit.Effects;
using PickerChangeColor = FormsCommunityToolkit.Effects.UWP.Effects.PickerChangeColor;
// or using PlatformEffects = FormsCommunityToolkit.Effects.UWP.Effects;

var color = (Color)Element.GetValue(RoutingEffects.PickerChangeColor.ColorProperty);


(similar for all the other classes)